### ISSUE TITLE
Graph: batch requests incorrectly formatted

### ISSUE THREAD

**Issue by RoelVB** [pnp/pnpjs#1085]
### Category
- [ ] Enhancement
- [x] Bug
- [ ] Question
- [ ] Documentation gap/issue

### Version

Please specify what version of the library you are using: [1.3.9]

Please specify what version(s) of SharePoint you are targeting: [Graph]

### Expected / Desired Behavior / Question
We should be able to add users to a group in a batch

### Observed Behavior
When adding a user in a batch the request looks like:
```
{"requests":[{"id":"1","method":"POST","url":"/groups/<SomeId>/members/$ref","body":"{\\"@odata.id\\":\\"https://graph.microsoft.com/v1.0/users/<SomeId>\\"}","headers":{}}]}
```
There is no `Content-Type` header in the request and if there was a correct `Content-Type` header the body should be JSON and not stringified JSON.

It's unclear to me how/where PnPJS generates this request.

### Steps to Reproduce
Try adding users to a group in a batch:
```typescript
graph.groups.getById('SomeID').members.inBatch(batch).add(`https://graph.microsoft.com/v1.0/users/SomeId`);
```

**Comment by patrick-rodgers** [comment]
Ok, batching worked in the past, so perhaps there have been changes. Will have to investigate.

**Comment by patrick-rodgers** [comment]
Fixed both of these issues, will go out in 2.0.4. Thanks!
